templates/default/delos.css

```diff
@@ -5910,7 +5910,7 @@ button.close {
   height: 10px;
   margin: 1px;
   text-indent: -999px;
-  border: 1px solid #ffffff;
+  border: 1px solid #fff;
```

Comment:
Yeah, not exactly a no change, but a change that is not about what this PR should do. The problem with this for me is, that it is hard to find the real change with this diff.